apps/ensrainbow/src/index.test.ts

```diff
@@ -9,13 +9,14 @@ import { app, db } from "./index";
 import { LABELHASH_COUNT_KEY } from "./utils/constants";
 
 describe("ENS Rainbow API", () => {
+  const port = 3223;
```

Comment:
Had to pick other port that would be less likely to conflict with other local services 🚀